umap/static/umap/js/umap.popup.js

```diff
@@ -116,12 +116,7 @@ L.U.PopupTemplate.Default = L.Class.extend({
 
   renderFooter: function () {
     if (this.feature.hasPopupFooter()) {
-      const footerContainer = L.DomUtil.create(
-          'div',
-          'umap-footer-container',
```

Comment:
You need to remove that class from base.css too

Reply:
:+1: